surveyform.html

```diff
@@ -0,0 +1,30 @@
+<h1  id='title'> title  </title></h1>
+<p id='description'> Description of the form<p>
+
+<form id="survey-form" >
+  
+    <input type="text" placeholder="Bleu Yves" name="model"><br /><br />
+  
+    <input id="email" placeholder="bleuyves@gmail.com" name="age"><br /><br />
+
+<input type="number" id="quantity" name="quantity" min="1" max="5">
+ <br /><br />   
+
+    <select name="cars" id="dropdown">
+  <option value="volvo">Volvo</option>
+  <option value="saab">Saab</option>
+  
+</select><br /><br />
+
+<input type="radio" id="html" name="fav_language" value="HTML">
+<label for="html">HTML</label><br>
+
+<input type="radio" id="html" name="fav_language" value="">
+<label for="html">CSS</label><br>
+
+<textarea id="w3review" name="w3review" rows="4" cols="50">
+Text area
+</textarea>
```

Comment:
Please indent your code, it will be clear and simple to understand when the lines will grow up.